src/Time/Tags.hpp

```diff
@@ -71,15 +70,12 @@ struct HistoryEvolvedVariables : db::PrefixTag, db::SimpleTag {
 
 /// \ingroup DataBoxTagsGroup
 /// \ingroup TimeGroup
-/// Prefix for TimeStepper boundary history
-///
-/// \tparam Key type identifying a boundary
-/// \tparam Tag tag for boundary variables
-template <typename Key, typename Tag, typename Hash = std::hash<Key>>
-struct HistoryBoundaryVariables : db::PrefixTag, db::SimpleTag {
-  static std::string name() noexcept { return "HistoryBoundaryVariables"; }
-  using tag = Tag;
-  using type = std::unordered_map<Key, db::item_type<Tag>, Hash>;
+/// Tag for TimeStepper boundary history
+template <typename LocalVars, typename RemoteVars, typename CouplingResult>
+struct BoundaryHistory : db::SimpleTag {
```

Comment:
It shouldn't be either, since the time steppers shouldn't know anything about domain decompositions.  There are a bunch of "boundary" things in the Time section of the code that should be changed, but I haven't decided on a name.  I've files bug #875.